tests/describe_task_queue_test.go

```diff
@@ -218,10 +160,18 @@ func (s *DescribeTaskQueueSuite) ValidateDescribeTaskQueue(tl string) *workflows
 		s.Assert().Equal(1, len(resp.GetVersionsInfo()), "should be 1 because only default/unversioned queue")
 		versionInfo := resp.GetVersionsInfo()[""]
 		s.Assert().Equal(enumspb.BUILD_ID_TASK_REACHABILITY_REACHABLE, versionInfo.GetTaskReachability())
-		return len(versionInfo.GetTypesInfo()) == 2
-
+		types := versionInfo.GetTypesInfo()
+		s.Assert().Equal(len(types), 2)
```

Comment:
once this is merged to the main branch, you will see that there is a third task type there, Nexus tasks.
```suggestion
		s.Assert().Equal(len(types), len(expectedBacklogCount))
```

Reply:
Oh, didn't know about Nexus. Changed, thanks.